pages/index.js

```diff
@@ -413,10 +413,14 @@ export default function Home({ total, today }) {
                       <div className="-mt-6">
                         <div>
                           <span className="inline-flex items-center justify-center rounded-xl bg-indigo-500 p-3 shadow-lg">
+                            <Link
```

Comment:
It'd be better to have the full span inside the link so that the whole icon div is clickable and not just the icon

see it's not clickable right there
![image](https://user-images.githubusercontent.com/95094057/212543778-cb4b678c-81c6-40f0-a77b-a00eb243674b.png)